front/app/modules/commercial/custom_topics/services/topics.ts

```diff
@@ -0,0 +1,24 @@
+import streams from 'utils/streams';
```

Comment:
These functions were originally in `services/topics`, but it turns out they were only used in the `custom_topics` module. So I moved them in there